pkg/kubectl/cmd/set/set_selector_test.go

```diff
@@ -283,6 +283,13 @@ func TestGetResourcesAndSelector(t *testing.T) {
 				MatchExpressions: []metav1.LabelSelectorRequirement{},
 			},
 		},
+		{
+			name:          "no resource and selector",
+			args:          args{args: []string{}},
+			wantResources: []string{},
+			wantErr:       false,
```

Comment:
If return an error here,  We can not output an exact error to make a difference between `kubectl set selector` and `kubectl set selector -f test.yaml`.  We would output the error when run Validate(). For example.
```
#./cluster/kubectl.sh set selector       
error: one or more resources must be specified as <resource> <name> or <resource>/<name>

#./cluster/kubectl.sh set selector -f test.yaml
error: one selector is required
```